database.php

```diff
@@ -27,13 +22,17 @@ public function __construct()
         $this->connectionString = $string;
     }
 
-    public function connectDB()
+    private function connectDB()
```

Comment:
i left my functions public but this `connectDB() `function. This one does return anything to be used outside of this class.  I believe all other classes return something to be displayed so I left them public.